<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesion</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../descargar.jpeg</normaloff>../../../../descargar.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff
</string>
  </property>
  <widget class="QPushButton" name="bttnInicio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>Inicio de sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small Semibold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000</string>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small Semibold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000
</string>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Usuario</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left: 10px;
color: #000;
border: none</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="placeholderText">
    <string>Usuario...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Password</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:10px;
border: none;
color: #000
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>35</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Contraseña..</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
